lib/engine/game/g_18_india/step/draft.rb

```diff
@@ -75,6 +75,8 @@ def process_bid(action)
             player.hand << company
             player.hand.sort_by! { |item| [item.name, -item.value] }
 
+            player.draft_history = player.draft_history + company.name + (company.type == :president ? '(Dir), ' : ', ')
```

Comment:
This is going to give a trailing comma (and space) on the draft history string. Instead you could define `draft_history` as an empty array in the `Player.initialize` method, then here add an item to the array:
```suggestion
            player.draft_history << company.name + (company.type == :president ? '(Dir)' : '')
```
And in the `Game.timeline` method join the array items together:
```
timeline << "#{p.name}: #{p.draft_history.join(', ')}"
```